components/cam/cime_config/config_component.xml

```diff
@@ -114,21 +114,21 @@
       <value compset="_CAM5%SP1V1-MAML"      >-use_MAML -crm_ny 1 -crm_ny_rad 1 </value>
       <value compset="_CAM5%SP1V1-MAML_"     >-crm_nx 64 -crm_nx_rad 64 </value>
       <!-- New MMF / super-parameterization compsets -->
-      <value compset="_CAM5%MMF[12]"           >-use_MMF -crm_adv MPDATA -nlev 72 -crm_nz 58 </value>
-      <value compset="_CAM5%MMF[12]"           >-crm_dx 1000 -crm_dt 5 </value>
-      <value compset="_CAM5%MMF[12]"           >-microphys mg2  </value>
-      <value compset="_CAM5%MMF[12]"           >-cppdefs ' -DMMF_DIR_NS ' </value>
-      <value compset="_CAM5%MMF[12]"           >-rad rrtmgp </value>
-      <value compset="_CAM5%MMF1"              >-MMF_microphysics_scheme sam1mom</value>
-      <value compset="_CAM5%MMF1"              >-chem none </value>
+      <value compset="_CAM5%MMF"               >-use_MMF -crm_adv MPDATA -nlev 72 -crm_nz 58 </value>
+      <value compset="_CAM5%MMF"               >-crm_dx 1000 -crm_dt 5 </value>
+      <value compset="_CAM5%MMF[12]"           >-crm sam</value>
+      <value compset="_CAM5%MMFXX"             >-crm samxx</value>
+      <value compset="_CAM5%MMF"               >-microphys mg2  </value>
+      <value compset="_CAM5%MMF"               >-cppdefs ' -DMMF_DIR_NS ' </value>
+      <value compset="_CAM5%MMF"               >-rad rrtmgp </value>
+      <value compset="_CAM5%MMF[1X]"           >-MMF_microphysics_scheme sam1mom -chem none</value>
       <value compset="_CAM5%MMF2"              >-MMF_microphysics_scheme m2005 </value>
       <value compset="_CAM5%MMF2"              >-chem linoz_mam4_resus_mom_soag -rain_evap_to_coarse_aero </value>
       <value compset="_CAM5%MMF2-ECPP"         >-use_ECPP </value>
       <value compset="_CAM5%MMF[12]-MAML"      >-use_MAML </value>
-      <value compset="_CAM5%MMF[12]"           >-crm_nx 64 -crm_nx_rad 16 -crm_ny 1 -crm_ny_rad 1 </value>
+      <value compset="_CAM5%MMF[12X]"          >-crm_nx 64 -crm_nx_rad 16 -crm_ny 1 -crm_ny_rad 1 </value>
       <value compset="_CAM5%MMF[12]-MAML"      >-crm_nx 64 -crm_nx_rad 64 -crm_ny 1 -crm_ny_rad 1 </value>
-      <value compset="_CAM5%MMF[12]-RCE"       >-aquaplanet -rce </value>
-      <!--  -->
+      <value compset="_CAM5%MMF[12X]-RCE"      >-aquaplanet -rce </value>
```

Comment:
There's likely a conflict here from the recent PR that added a MMF aqua planet compset. I can fix this after you rebase.